modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -125,6 +129,7 @@ NOTE: The `mcrypt` library is deprecated and removed from the PHP 7.2 core. Howe
 === php-libsodium
 
 If you see the message `PHP Fatal error: sodium_init() in Unknown on line 0` when invoking the command `php -v` or when running PHP programs, `php-libsodium` might be installed.
+From PHP 7.2 onwards, libsodium is part of PHP, loading the library is no longer necessary.
```

Comment:
This would be more obvious to the user if it was formatted as an admonition.

Reply:
on the way to fix this and the above

Reply:
fixed